styles/style.css

```diff
@@ -13,3 +13,4 @@
 @import url("../components/input/input.css");
 @import url("../components/text-utilities/text-util.css");
 @import url("../components/lists/lists.css");
+@import url("../components/navigation/navigation.html");
```

Comment:
Here you shall mention navigation.css instead of .html

Reply:
Thanks for noticing that.